app/models/lookup_value.rb

```diff
@@ -19,14 +20,12 @@ class LookupValue < ActiveRecord::Base
   serialize :value
   attr_name :match
 
-  scope :default, -> { where(:match => "default").limit(1) }
-
   scoped_search :on => :value, :complete_value => true, :default_order => true
   scoped_search :on => :match, :complete_value => true
   scoped_search :in => :lookup_key, :on => :key, :rename => :lookup_key, :complete_value => true
 
   def value_present?
-    self.errors.add(:value, :blank) if value.to_s.empty? && !use_puppet_default && lookup_key.puppet?
+    self.errors.add(:value, :blank) if value.to_s.empty? && !use_puppet_default && lookup_key.puppet? && match != 'default'
```

Comment:
Can default value for SmartClassParameter be empty?
The error won't be raised in this case.